tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+      "src/**/*",
+      "test/**/*"
```

Comment:
I think you can (and should) drop this line. There's already `src/test` and we'll remove the `test` symlink soon.

Reply:
Same for `tsconfig.test.json`, there you could also drop the `test` include.

And then, you can actually drop the whole `"include"` in `tsconfig.{build,test}.json` because the default `tsconfig.json` already includes only `src/**/*`. You can also remove the empty `"exclude"` from test.json because the main one already has an empty exclude.